packages/amplify-category-hosting/lib/S3AndCloudFront/helpers/configure-Website.js

```diff
@@ -8,22 +8,39 @@ async function configure(context) {
       type: 'input',
       message: 'index doc for the website',
       default: WebsiteConfiguration.IndexDocument,
+      validate: docKeyValidation,
     },
     {
       name: 'ErrorDocument',
       type: 'input',
       message: 'error doc for the website',
       default: WebsiteConfiguration.ErrorDocument,
+      validate: docKeyValidation,
     },
   ];
 
   const answers = await inquirer.prompt(questions);
-  WebsiteConfiguration.IndexDocument = answers.IndexDocument;
-  WebsiteConfiguration.ErrorDocument = answers.ErrorDocument;
+  WebsiteConfiguration.IndexDocument = answers.IndexDocument.trim();
+  WebsiteConfiguration.ErrorDocument = answers.ErrorDocument.trim();
 
   return context;
 }
 
+function docKeyValidation(str) {
+  str = str.trim();
+
+  let isValid = str.length > 0;
+  if (!isValid) {
+    return 'Must not be empty, or only contains space charactors.';
+  }
+
+  isValid = !/\//.test(str);
+  if (!isValid) {
+    return 'The slash charactor is not allowed.';
```

Comment:
Is backslash allowed?

Reply:
backslash is actually allowed, although probably very ugly.